src/logic/graphLabeling.ts

```diff
@@ -0,0 +1,68 @@
+import {
+  GraphIndex,
+  MappedGraph,
+  MappedEdge,
+  MappedNode,
+  LabeledGraph,
+  LabeledEdge,
+  LabeledNode,
+} from './commonInterfaces'
+
+const LABEL_ALPHABET = 'ABCDEFGHIJKLMNOPQRSTUVWXYZ';
+
+export default function labelGraph(graph: MappedGraph): LabeledGraph {
+  return [
+    labelNodes(graph[GraphIndex.NODES]),
+    labelEdges(graph[GraphIndex.EDGES]),
+  ];
+}
+
+/**
```

Comment:
This function has 2 lines. The docstring has 4. You can easily make the function more descriptive and remove the docstring.